bin/cask

```diff
@@ -236,26 +236,48 @@ def find_best_emacs():
     else:
         return 'emacs'
 
+def inside_emacs_24():
+    """Whether Cask is running inside Emacs-24.
+
+Emacs 24 is rather inconsistent in its use of the ``$EMACS`` and
+``$INSIDE_EMACS`` environment variables, so we try here to test if Cask is
+being run inside Emacs (either in a shell or with ``M-x compile``).
+
+Return ``True`` when running in Emacs 24, ``False`` otherwise.
+    """
+    ## are we inside Emacs at all
+    return (b'INSIDE_EMACS' in ENVB and
+            ## M-x compile in Emacs-24 sets INSIDE_EMACS to exactly 't'
+            (ENVB.get(b'INSIDE_EMACS') == 't'
+             or
```

Comment:
@phillord Uhm, I'm sorry but don't you see that my comment is associated with a line?!  Specifically the line right above my comment?!  I'm sorry but I thought that Github line comments were a known thing…